src/server/config.ts

```diff
@@ -81,12 +81,16 @@ export interface DataRequestConfig extends AxiosRequestConfig {
       | ApiErrorResponse<null>
       | ApiPostponeResponse
   ) => any;
+
+  page?: number;
 }
 
 const ONE_SECOND_MS = 1000;
 const ONE_MINUTE_MS = 60 * ONE_SECOND_MS;
 const ONE_HOUR_MS = 60 * ONE_MINUTE_MS;
 
+export const MAXIMUM_AMOUNT_OF_PAGES = 10; // If we are requesting a paginated rescource request a maximum of 10 pages.
```

Comment:
10 is wel wat arbitrair, geen idee hoeveel data dat is uiteindelijk. Is het mogelijk dit variabel te maken in de dataset configs?

Reply:
Check, heb de variabele verplaatst naar de config.